CharacterGenerator/CharacterGenerator/Manager.m

```diff
@@ -0,0 +1,38 @@
+//
+//  Manager.m
+//  CharacterGenerator
+//
+//  Created by Z on 8/6/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "Manager.h"
+
+@implementation Manager
+
+//singleton code
++ (instancetype)sharedInstance
+{
+    static Manager *sharedInstance = nil;
```

Comment:
Good to know this. I still don't understand fully about how exactly singleton and delegation works and their differences. I just use the syntax even though I don't get it. Derek is good at this though. He's done great with our project.